index.html

```diff
@@ -0,0 +1,30 @@
+<!--HTML layout page-->
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" type="text/css">
+    <title>Portfolio | Sully</title>
+</head>
+<body class="main-content">
+    <header class="mainheader">

```

Comment:
- [ ] **OPTIONAL**
I recommend referring to the unrepeatable tags with their actual names in the CSS because there is only one `<body>` tag and one `<header>` tag on any website so, it doesn't make sense to attach a `class` attribute to them right? Tell me what you think!